### PR TITLE
feat: add seach input to candy machines top actions list

### LINKED ISSUE #92: [Task]: Add seach input to candy machines top actions list
**Task description.**

Add seach input to candy machines top actions list
image

**Requirements.**
- UI is correct
- Functinality works

### PR DESCRIPTION
**Pull request by VmMad**
Fixes #92 

## Changes

  - Add Recoil and Magnifying Glass svg for search input
  - Create Atoms file, TopActions and Search Bar components
  - Remove input from the CandyMachineCard
  
### Checks

* [x] Have you catchup your branch with the latest state in base?
* [x] Have you lint your code locally prior to submission?
* [x] Have you reviewed your proposed changes and removed debris?
* [x] Have you successfully ran due tests with your changes?

### CODE REVIEW

**Review by evavirseda — CHANGES_REQUESTED** [review]
Looks really great Jean :rocket: , just one thing, imo Searchbar should not be part of App, because there are some routes that do not show it

image

image

**Comment by VmMad** [comment]
> Looks really great Jean :rocket: , just one thing, imo Searchbar should not be part of App, because there are some routes that do not show it
> 
> image
> 
> image

I thought of putting it in the app because if we put it in all the components it's going to be kinda redundant. I was thinking of ways to do it more cleaner and I thought maybe with it being conditional depending on the route it can behave differently in each case. What do you think?

**Review by evavirseda — APPROVED** [review]
Looks great Jean :rocket:

File: components/SearchBar.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const SearchBar: FC<{
+    searchValue: string
+    setSearchValue: SetterOrUpdater<string> | Dispatch<SetStateAction<string>>
+}> = ({ searchValue, setSearchValue }) => {
+    return (
+        <TextInput
+            icon={SearchIcon}
+            className='border py-1 rounded-lg searchbox-candymachine f5 pl-2'
+            type='search'
+            placeholder='Search candy machine'
```
Comment (cpl121) [inline components/SearchBar.tsx:15]: As the component is generic, I think that the text of the placeholder should be a prop

Comment (VmMad) [reply components/SearchBar.tsx:15]: You are right. I didn't think about that

Comment (cpl121) [reply components/SearchBar.tsx:15]: This needs to be resolved

**Review by cpl121 — APPROVED** [review]
Amazing work! 💯

**Pull request merged**